tests/unittest_odgovlt.py

```diff
@@ -0,0 +1,57 @@
+from ckanext.harvest.harvesters.ckanharvester import CKANHarvester
+from ckanext.harvest.harvesters.ckanharvester import DatetimeEncoder
+from mock import patch
+from sqlalchemy import MetaData
+from sqlalchemy.engine.base import Engine
+from datetime import datetime
+
+ckan_object = CKANHarvester()
+datetime_object = DatetimeEncoder()
+
+
+def test_info():
+    info = ckan_object.info()
+    assert isinstance(info, dict)
+    dict_names = [
+           'name',
+           'title',
+           'description',
+           'form_config_interface']
+    for dict_name in info:
+        assert isinstance(info[dict_name], str)
+        assert isinstance(dict_name, str)
+        assert dict_name in dict_names
```

Comment:
I don't understand meaning of these two lines. And usually, when writing tests, DRY principle in most cases do not apply, to instead of testing all items in the for loop, you could simple do multiple asserts as you did in the line below. And this will bu much more readable.

And usually there is no need to test someone's else code, test only you own code.